account_reconcile_compassion/models/bank_statement_line.py

```diff
@@ -155,6 +62,7 @@ def get_statement_line_for_reconciliation_widget(self):
     ##########################################################################
 
     @api.multi
+    @job(default_channel="root.bank_reconciliation")
     def process_reconciliation(
```

Comment:
You could rename it to `_process_reconciliation` and add another method `process_reconciliation` that calls `self.with_delay()._process_reconciliation()` in order to force this method to always run in a separate thread.